site/docs/src/json/tenants/search-response.json

```diff
@@ -0,0 +1,346 @@
+{
+  "tenants": [
+    {
+      "accessControlConfiguration": {
+        "uiIPAccessControlListId": "11d49de7-69f6-46fc-8270-0b3aa626327a"
+      },
+      "configured": true,
+      "captchaConfiguration": {
+        "captchaMethod": "GoogleRecaptchaV3",
+        "enabled": false,
+        "threshold": 0.5
+      },
+      "connectorPolicies": [
+        {
+          "connectorId": "e3306678-a53a-4964-9040-1c96f36dda72",
+          "domains": [
+            "*"
+          ],
+          "migrate": false
+        },
+        {
+          "connectorId": "27f22280-7e55-4d1c-b9f8-239bf9cc1a5e",
+          "domains": [
+            "*"
+          ],
+          "migrate": true
+        }
+      ],
+      "data": {
+        "description": "No more secrets, Marty."
```

Comment:
Does it make sense to be able to search on `data`? I could see that being useful. ES search would be best, but if not that, even plain text fuzzy search would be good.

(Same for applications now that I think about it.)

Reply:
Per internal discussion, foregoing this right now.